_docs/kustomize/akash-provider/run.sh

```diff
@@ -1,24 +1,31 @@
 #!/bin/sh
 
-set -e
+set -xe
```

Comment:
Do we need to keep `-x` here? Usually makes CI more verbose than necessary.

Reply:
Yes I can remove this.